source/vscode/README.md

```diff
@@ -1,42 +0,0 @@
-## Features
```

Comment:
Can you keep this file, and add something like `vscode is removed, and please goto link to find the source code`

Reply:
That's documented in the extension page itself: https://marketplace.visualstudio.com/items?itemName=madewithcardsio.adaptivecardsstudiobeta

I don't think we should be responsible for keeping a readme up-to-date for that as we don't own it and that repo could change

Reply:
You are right. look like the old extension is deleted, so we should not have link to this